content/post/data-warehouse-sla-p2.md

```diff
@@ -27,47 +28,61 @@ Here are four examples of services provided with increasing specificity:
 
 
 ### Expected performance
-Imagine you find a magic data genie and you ask for the most detailed sales data in the world. You start flipping through the data and everything looks AMAZING... until you realize the data is two years old. That wish could have used a detailed definition of expected performance. This is the area of the SLA where we move from "what" will be delivered to "how" it will be delivered.
+Imagine you find a magic data genie and you ask for the most detailed sales data in the world. You start flipping through the data and everything looks AMAZING... until you realize the data is two years old. That wish could have used a definition of expected performance. This is the area of the SLA where we move from "what" will be delivered to "how" it will be delivered.
 
 Here are some common aspects of expected performance to include in your SLA:
 * Data freshness: How current is the data? Is the data live, updated every hour or ready at 7am every morning? A potential approach is to categorize KPIs / models / data sources using "tiers of freshness" (e.g., intraday, daily, weekly) 
 * Query speed: Query speed is a challenging aspect to quantify as your data warehouse most likely supports several different use cases in your data warehouse (types of data and queries). Average query run time is a good place to start, but you may need to create a more nuanced metric (e.g., X% of queries finish in <Y seconds)
```

Comment:
This sentence reads a bit odd, probably because of the two occurrences of `data warehouse`. Would it still work if you ended it `...different use cases (types of data and queries).`